<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-26T13:38:22 IST" name="Suite" finished-at="2024-05-26T13:38:42 IST" duration-ms="19562">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.run@28cda624]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-05-26T13:38:22 IST" name="Test" finished-at="2024-05-26T13:38:42 IST" duration-ms="19562">
      <class name="TestRunner.run">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.run@28cda624]" started-at="2024-05-26T13:38:22 IST" name="setUpClass" finished-at="2024-05-26T13:38:23 IST" duration-ms="431" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@2026fbff]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.run@28cda624]" started-at="2024-05-26T13:38:23 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-05-26T13:38:31 IST" duration-ms="7838" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful Login with Valid Credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.run@28cda624]" started-at="2024-05-26T13:38:31 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-05-26T13:38:38 IST" duration-ms="7527" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful Login with Valid Credential DDT"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.run@28cda624]" started-at="2024-05-26T13:38:38 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-05-26T13:38:42 IST" duration-ms="3451" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful Login with Valid Credential DDT"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at StepDefination.stepdef.page_title_should_be(stepdef.java:148)
at âœ½.Page Title should be "Dashboard / nopCommerce administration"(file:///A:/eclipsCucumber_workspace/firstProject_bdd/./Features/loginfeature.feature:20)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.run@28cda624]" started-at="2024-05-26T13:38:42 IST" name="tearDownClass" finished-at="2024-05-26T13:38:42 IST" duration-ms="259" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.run -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
